tests/examples/source/devfilesV2/nodejs/devfileSourceMapping.yaml

```diff
@@ -0,0 +1,40 @@
+schemaVersion: 2.0.0
+metadata:
+  name: nodejs
```

Comment:
I can see only devfileSourceMapping.yaml for `nodejs` component type. So i think this pr only handles for `nodejs` type and later other supported devfile component type will be added, right ?

Reply:
The sample devfile I used is nodejs, but the implementation of `sourceMapping` isn't specific to any devfile or runtime, so other devfiles don't need to be added, as it would be testing the same functionality.